contrib/packaging/docker/Dockerfile.runtime

```diff
@@ -9,16 +9,29 @@ apt-get upgrade -y && \
 #
 apt-get install -y --no-install-recommends \
     gpg gpg-agent libelf-dev libmnl-dev libc6-dev-i386 iptables libgcc-5-dev \
-    bash-completion binutils binutils-dev ca-certificates && \
+    bash-completion binutils binutils-dev ca-certificates clang-7 llvm-7 && \
 apt-get purge --auto-remove && \
-apt-get clean
+apt-get clean && \
+#
+# clang-7
```

Comment:
What are the effects of doing this here vs. in runtime-build?

Reply:
runtime-build doesn't operate apt-get with updated/upgraded repositories, so this just seemed like the quickest way to get the up to date version installed into the runtime image (ie without having to also change runtime-build to do update/upgrade).

runtime-base is the basis for all tools we can install via apt that we will need in the final image.

runtime-build is for building the dependencies we'll need in the final image, which typically requires basic tools + links to the specific version of software we want to compile, which we then copy over. The apt dependencies there do not strictly need to be the latest versions.